feature-libs/checkout/base/root/facade/checkout-delivery-address.facade.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Address, facadeFactory, QueryState } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { CHECKOUT_CORE_FEATURE } from '../feature-name';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: CheckoutDeliveryAddressFacade,
+      feature: CHECKOUT_CORE_FEATURE,
+      methods: [
+        'getDeliveryAddressState',
+        'createAndSetAddress',
+        'setDeliveryAddress',
+        'clearCheckoutDeliveryAddress',
+      ],
+      async: true,
```

Comment:
I believe we can delete all async property from here and other checkout facades as well as it's not using ngrx anymore, but super duper c&q right?

even though some just dispatches old ngrx actions

Reply:
Hmm, I'd keep it, as these facades still inject `store`. I added todos to remove it once we get rid of the store.